python/python/lance/vector.py

```diff
@@ -0,0 +1,84 @@
+"""Embedding vector utilities"""
+from typing import Union, Optional
+
+import numpy as np
+import pyarrow as pa
+
+#  Copyright 2023 Lance Developers
```

Comment:
License above imports.

Reply:
fixed

Reply:
Docstring is still on top of license. The file should start with license